pkg/operator/configobservation/scheduler/observe_scheduler.go

```diff
@@ -82,6 +82,10 @@ func ObserveSchedulerConfig(genericListers configobserver.Listers, recorder even
 		sourceTargetLocation,
 	)
 	if len(configMapName) == 0 {
+		if len(currentPolicyConfigMapName) > 0 {
+			recorder.Eventf("ObservedConfigMapNameChanged", "scheduler configmap removed")
```

Comment:
Can you update the existing unit test or add e2e for this scenario?

Reply:
updated